docs/whatsnew-4.4.rst

```diff
@@ -0,0 +1,531 @@
+Jeremy Cohen <jcohen02@users.noreply.github.com>
+John Arnold <johnar@microsoft.com>
+Jon Banafato <jonathan.banafato@gmail.com>
+Jon Dufresne <jon.dufresne@gmail.com>
+Joshua Engelman <j.aaron.engelman@gmail.com>
+Joshua Schmid <jschmid@suse.com>
+Josue Balandrano Coronel <xirdneh@gmail.com>
+K Davis <anybodys@users.noreply.github.com>
+kidoz <ckidoz@gmail.com>
+Kiyohiro Yamaguchi <kiyoya@gmail.com>
+Korijn van Golen <korijn@gmail.com>
+Lars Kruse <devel@sumpfralle.de>
+Lars Rinn <lm.rinn@outlook.com>
+Lewis M. Kabui <lewis.maina@andela.com>
+madprogrammer <serg@anufrienko.net>
+Manuel Vázquez Acosta <mvaled@users.noreply.github.com>
+Marcus McHale <marcus.mchale@nuigalway.ie>
+Mariatta <Mariatta@users.noreply.github.com>
+Mario Kostelac <mario@intercom.io>
+Matt Wiens <mwiens91@gmail.com>
+Maximilien Cuony <the-glu@users.noreply.github.com>
+Maximilien de Bayser <maxdebayser@gmail.com>
+Meysam <MeysamAzad81@yahoo.com>
+Milind Shakya <milin@users.noreply.github.com>
+na387 <na387@users.noreply.github.com>
+Nicholas Pilon <npilon@gmail.com>
+Nick Parsons <nparsons08@gmail.com>
+Nik Molnar <nik.molnar@consbio.org>
+Noah Hall <noah.t.hall@gmail.com>
+Noam <noamkush@users.noreply.github.com>
+Omer Katz <omer.drow@gmail.com>
+Paweł Adamczak <pawel.ad@gmail.com>
+peng weikang <pengwk2@gmail.com>
+Prathamesh Salunkhe <spratham55@gmail.com>
+Przemysław Suliga <1270737+suligap@users.noreply.github.com>
+Raf Geens <rafgeens@gmail.com>
+(◕ᴥ◕) <ratson@users.noreply.github.com>
+Robert Kopaczewski <rk@23doors.com>
+Samuel Huang <samhuang91@gmail.com>
+Sebastian Wojciechowski <42519683+sebwoj@users.noreply.github.com>
+Seunghun Lee <waydi1@gmail.com>
+Shanavas M <shanavas.m2@gmail.com>
+Simon Charette <charettes@users.noreply.github.com>
+Simon Schmidt <schmidt.simon@gmail.com>
+srafehi <shadyrafehi@gmail.com>
+Steven Sklar <sklarsa@gmail.com>
+Tom Booth <thomasbo@microsoft.com>
+Tom Clancy <ClancyTJD@users.noreply.github.com>
+Toni Ruža <gmr.gaf@gmail.com>
+tothegump <tothegump@gmail.com>
+Victor Mireyev <victor@opennodecloud.com>
+Vikas Prasad <vikasprasad.prasad@gmail.com>
+walterqian <walter@color.com>
+Willem <himself@willemthiart.com>
+Xiaodong <xd_deng@hotmail.com>
+yywing <386542536@qq.com>
+
+.. note::
+
+    This wall was automatically generated from git history,
+    so sadly it doesn't not include the people who help with more important
+    things like answering mailing-list questions.
+
+
+Upgrading from Celery 4.3
+=========================
+
+Please read the important notes below as there are several breaking changes.
+
+.. _v430-important:
+
+Important Notes
+===============
+
+Supported Python Versions
+-------------------------
+
+The supported Python Versions are:
+
+- CPython 2.7
+- CPython 3.5
+- CPython 3.6
+- CPython 3.7
+- CPython 3.8
+- PyPy2.7 7.2 (``pypy2``)
+- PyPy3.5 7.1 (``pypy3``)
+- PyPy3.6 7.2 (``pypy3``)
+
+Kombu
+-----
+
+Starting from this release, the minimum required version is Kombu 4.6S.
```

Comment:
We'll probably release Kombu 4.6.6 and use that as a minimum version.

Reply:
true.